salt/roots/salt/memex-explorer.sls

```diff
@@ -80,6 +80,8 @@ celery:
     - name: /home/vagrant/miniconda/envs/memex/bin/celery --detach --loglevel=debug --logfile=/vagrant/source/celeryd.log --workdir="/vagrant/source" -A memex worker
     - cwd: /vagrant/source
     - user: vagrant
+    - env:
+        - JAVA_HOME: '/usr/lib/jvm/java-7-oracle'
```

Comment:
@amfarrell - Here's an example of setting an environment variable and other customizations in a "service" command.  The `--detach` flag here is how this is getting daemonized.